submitted_models/coro_karen_sensor_config_1/model.sdf

```diff
@@ -0,0 +1,1347 @@
+        <geometry>
+          <cylinder>
+            <length>0.011</length>
+            <radius>0.005</radius>
+          </cylinder>
+        </geometry>
+        <material>
+          <ambient>0.0 0.0 0.0 1.0</ambient>
+          <diffuse>0.0 0.0 0.0 1.0</diffuse>
+        </material>
+      </visual>
+
+      <visual name="camera_rear_visual">
+        <pose>0.28 0 0.645 0 0 0</pose>
+        <geometry>
+          <box>
+            <size>0.01 0.03 0.03</size>
+          </box>
+        </geometry>
+        <material>
+          <ambient>0.0 0.0 0.0 1.0</ambient>
+          <diffuse>0.0 0.0 0.0 1.0</diffuse>
+        </material>
+      </visual>
+      <visual name="camera_lens_rear_visual">
+        <pose>0.27 0 0.645 0 1.5707963267948966 0</pose>
+        <geometry>
+          <cylinder>
+            <length>0.011</length>
+            <radius>0.005</radius>
+          </cylinder>
+        </geometry>
+        <material>
+          <ambient>0.0 0.0 0.0 1.0</ambient>
+          <diffuse>0.0 0.0 0.0 1.0</diffuse>
+        </material>
+      </visual>
+
+      <visual name="camera_left_visual">
+        <pose>0.37 0.09 0.645 0 0 0</pose>
+        <geometry>
+          <box>
+            <size>0.03 0.01 0.03</size>
+          </box>
+        </geometry>
+        <material>
+          <ambient>0.0 0.0 0.0 1.0</ambient>
+          <diffuse>0.0 0.0 0.0 1.0</diffuse>
+        </material>
+      </visual>
+      <visual name="camera_lens_left_visual">
+        <pose>0.37 0.1 0.645 1.5707963267948966 0 0</pose>
+        <geometry>
+          <cylinder>
+            <length>0.011</length>
+            <radius>0.005</radius>
+          </cylinder>
+        </geometry>
+        <material>
+          <ambient>0.0 0.0 0.0 1.0</ambient>
+          <diffuse>0.0 0.0 0.0 1.0</diffuse>
+        </material>
+      </visual>
+
+      <visual name="camera_right_visual">
+        <pose>0.37 -0.09 0.645 0 0 0</pose>
+        <geometry>
+          <box>
+            <size>0.03 0.01 0.03</size>
+          </box>
+        </geometry>
+        <material>
+          <ambient>0.0 0.0 0.0 1.0</ambient>
+          <diffuse>0.0 0.0 0.0 1.0</diffuse>
+        </material>
+      </visual>
+      <visual name="camera_lens_right_visual">
+        <pose>0.37 -0.1 0.645 1.5707963267948966 0 0</pose>
+        <geometry>
+          <cylinder>
+            <length>0.011</length>
+            <radius>0.005</radius>
+          </cylinder>
+        </geometry>
+        <material>
+          <ambient>0.0 0.0 0.0 1.0</ambient>
+          <diffuse>0.0 0.0 0.0 1.0</diffuse>
+        </material>
+      </visual>
+
+
+      <!-- Real robot has 2 x B0203 with LN008 each with a 180 deg hor x 130 deg vert FOV
+           Since fisheye cameras are not modeled, four VGA cameras are used in the model -->
+      <sensor name="camera_front" type="camera">
+        <pose>0.48 0.06 0.645 0 0 0</pose>
+        <always_on>1</always_on>
+        <update_rate>30</update_rate>

```

Comment:
The specification sheet indicates shit sensor should be 25 Hz.

Reply:
Fixed.